src/mappings/FundingRequest/schema.graphql

```diff
@@ -0,0 +1,8 @@
+type FundingRequestProposal @entity {
+	id: ID!
```

Comment:
should it include also if its token or native coin ?
if token , so its address ?

Reply:
It already includes it in the scheme parameters. No need to cache this per proposal as it is the same for all proposals of the scheme